README.rst

```diff
@@ -171,6 +171,9 @@ Parse URL
     - ``Protocol``  "http"
         Use 'Hyper Text Transfer Protocol'
 
+    - ``Domain``  "google.com"
+        Connect to the server at 'google.com'
+
     - ``Resource``  "/"
         Retrieve main (index) page
 
```

Comment:
It can be useful to add this image:
![schema](https://asir.gonzaleztroyano.es/projects/iaw/_images/cp-5.png)
Source: https://en.wikipedia.org/wiki/File:URI_syntax_diagram.svg